geckomat/get_enzyme_data/flattenCell.m

```diff
@@ -0,0 +1,43 @@
+function Cflat = flattenCell(C,strFlag)

```

Comment:
It's coded by Jon, initially for HMR but scheduled to be part of [RAVEN](https://github.com/SysBioChalmers/RAVEN/pull/168/commits/d992169e9da44ed9c441753bcf77cc83f521e953). When merged in RAVEN, it can again be removed from here. How do you propose I document this in the function?

Reply:
@edkerk I included in [`a0146ee`](https://github.com/SysBioChalmers/GECKO/pull/49/commits/a0146ee12fee7579e7fe675d8187ca3006181687) a description of this :) that way we will remember to remove it in the future :)